src/zope/app/apidoc/interface.rst

```diff
@@ -227,7 +224,7 @@ This function returns a page-template-friendly dictionary for a method:
 
   >>> pprint(interface.getMethodInfoDictionary(IFoo['blah'])) #doc
   {'doc':
-     u'<p>This is the <cite>blah</cite> method.</p>\n',
+     u'<p>This is the `blah` method.</p>\n',
```

Comment:
As far as I can tell, something changed in one of the lower layers, likely `docutils`.